float32 pitch
float32 yaw
float32 zoom
float32 focus
char  home
char  TakePicture
char  cameraModeChange
